### ISSUE TITLE
How to install vmprof on macOS

### ISSUE THREAD

**Issue by traff** [vmprof/vmprof-python#149]
Now the Basic Usage section has subsections about how to install vmprof on Linux and Windows, but macOS is absent, which makes an impression that it is not supported.

**Comment by planrich** [comment]
Indeed. I have change the section and simplified it. Basic Usage is not 'Installation'
